From 21fc0b94f7981bc3bc1450835abd5534723072c1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20D=C3=B6singer?= <stefan@codeweavers.com>
Date: Thu, 21 Aug 2014 22:47:58 +0200
Subject: wined3d: Destroy views through the CS.

Move this ahead. This has caused sporadic test failures ever since blits were moved to the CS.
---
 dlls/wined3d/view.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/dlls/wined3d/view.c b/dlls/wined3d/view.c
index b4bc771..2147cc7 100644
--- a/dlls/wined3d/view.c
+++ b/dlls/wined3d/view.c
@@ -48,12 +48,6 @@ ULONG CDECL wined3d_rendertarget_view_decref(struct wined3d_rendertarget_view *v
     {
         struct wined3d_device *device = view->resource->device;
 
-        if (wined3d_settings.cs_multithreaded)
-        {
-            FIXME("Waiting for cs.\n");
-            device->cs->ops->finish(device->cs);
-        }
-
         /* Call wined3d_object_destroyed() before releasing the resource,
          * since releasing the resource may end up destroying the parent. */
         view->parent_ops->wined3d_object_destroyed(view->parent);
-- 
2.8.0

